Gaijinizer patch/Map470.txt

```diff
@@ -1,181 +1,227 @@
+here. I only plan on informing them of what they
+want to know.#Ron
 > END STRING
 
 > BEGIN STRING
 でも、せっかく各世界から集まってくれたんだ。
 
 座ってお茶でも飲みながら
 順を追って説明していこうじゃないか………
-> CONTEXT: Map470/events/1/pages/0/55/Dialogue/enemy27 < UNTRANSLATED
-
+> CONTEXT: Map470/events/1/pages/0/55/Dialogue/enemy27
+You've all gone to the trouble of gathering here
+from each of your worlds. Why don't you all sit,
+have some tea or something, and I'll explain
+everything step-by-step.#RonIffyWording
 > END STRING
 
 > BEGIN STRING
 ち、チェシャ……
-> CONTEXT: Map470/events/1/pages/0/61/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map470/events/1/pages/0/61/Dialogue/kazuya
+C-Cheshire...#Ron
 > END STRING
 
 > BEGIN STRING
 ………嬉しいよ。
 
 キミはまだ、ボクをそう呼んでくれるんだね……
-> CONTEXT: Map470/events/1/pages/0/68/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map470/events/1/pages/0/68/Dialogue/enemy29
+It makes me happy to hear you call me by that
+name...#Ron
 > END STRING
 
 > BEGIN STRING
 でも、今だけは……
 ボクを『マッドハッター』でいさせて欲しいんだ。
 
 『チェシャ』は、キミの思い出の中に居させて欲しい。
-> CONTEXT: Map470/events/1/pages/0/72/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map470/events/1/pages/0/72/Dialogue/enemy29
+But this one time, I'd like you to let me be
+"Mad Hatter". I want you to keep your memory of
+"Cheshire" deep inside.
 > END STRING
 
 > BEGIN STRING
 あの姿は、キミに好かれるための姿なんだよ……
 だから、今だけは……
 
 お願いできるかい？\\N[1]くん。
-> CONTEXT: Map470/events/1/pages/0/77/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: Map470/events/1/pages/0/77/Dialogue/enemy29
+I took that form so I'd be pleasing to you.
 
+So just once, can I ask this of you,
+\\N[1]-kun?#Ron
 > END STRING
 
 > BEGIN STRING
 あやつが、『マッドハッター』……
 
 我々を呼んだ張本人というわけだな…
-> CONTEXT: Map470/events/1/pages/0/97/Dialogue/alice_fc5 < UNTRANSLATED
+> CONTEXT: Map470/events/1/pages/0/97/Dialogue/alice_fc5
+That's Mad Hatter...
 
+So that's the culprit behind us being summoned.#Ron
 > END STRING
 
 > BEGIN STRING
 独特な雰囲気の悪魔だけど……
 少なくとも、禍々しい感じはしないね。
 
 むしろ、優しそうな感じがする……
-> CONTEXT: Map470/events/1/pages/0/101/Dialogue/enemy15 < UNTRANSLATED
+> CONTEXT: Map470/events/1/pages/0/101/Dialogue/enemy15
+That demon has quite a peculiar atmosphere about
+her, but it's not sinister.
 
+In fact, she actually seems quite nice.#Ron
 > END STRING
 
 > BEGIN STRING
 というより……
 \\N[1]と声が同じように思えるのだが…
 
 ヤツはどういう魔物なのだ…？
-> CONTEXT: Map470/events/1/pages/0/106/Dialogue/enemy35 < UNTRANSLATED
-
+> CONTEXT: Map470/events/1/pages/0/106/Dialogue/enemy35
+In fact, it seems both \\N[1] and her have
+rather similar voices. Just what kind of monster
+is she exactly?#Ron
 > END STRING
 
 > BEGIN STRING
 いや、私の召喚機にもバグったような反応してるから
 魔物か悪魔かどうかも怪しいわよ……
 
 そもそも、アイツって生命体なの……？
-> CONTEXT: Map470/events/1/pages/0/111/Dialogue/enemy38 < UNTRANSLATED
-
+> CONTEXT: Map470/events/1/pages/0/111/Dialogue/enemy38
+No, my summoning device is giving some really
+buggy responses, so it's hard to tell whether
+she's a monster or a demon. In fact, is she
+even a living organism?#RonIffy
 > END STRING
```

Comment:
Rephrasing for flow (let me know if this is a bit too far from the original text).
```
No... my summoner is throwing a bunch of errors.
It's hard to tell whether she's even a monster
or a demon... Is... Is she even a living being in
the first place...?
```